docker/apache-kafka/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM openjdk:11.0.6-jre-buster
+
+# build time
+ARG ZOOKEEPER_ADDRESS=zookeeper:2181
+
+# runtime
+ENV KAFKA_TAR="kafka_2.13-3.3.1.tgz"
```

Comment:
NIT: can we double check the image size of this docker image built from src vs the dockerhub image size, perhaps by bitnami?

Reply:
Would like to have our own docker file.